packages/customer/src/services/customer-module.ts

```diff
@@ -64,18 +72,191 @@ export default class CustomerModuleService implements ICustomerModuleService {
     )
   }
 
-  @InjectManager("baseRepository_")
   async create(
     data: CustomerTypes.CreateCustomerDTO,
+    sharedContext?: Context
+  ): Promise<CustomerTypes.CustomerDTO>
+
+  async create(
+    data: CustomerTypes.CreateCustomerDTO[],
+    sharedContext?: Context
+  ): Promise<CustomerTypes.CustomerDTO[]>
+
+  @InjectManager("baseRepository_")
```

Comment:
todo: Should use `InjectTransactionManager`

Reply:
Alright - when should you use one over the other?

Reply:
when the entry is a mutation you can use the transaction manager otherwise the manager

Reply:
Wouldn't this only be relevant in cases where a method does multiple mutations though?

Reply:
it is a best practice, otherwise you also need to manager the persist and flush manually which is not the case here